File: P2A2 Algoritmos a Lenguajes/sumas_pares.py
#Emilia Cristina Prieto Henriquez
#ID 00580909
#Ing. de Animacion Digital

num1= int(input("Ingrese el primer numero: "))

if num1 % 2 == 0:
    if num1 > 0:
        num2 = int(input("Ingrese el segundo número: "))
        
        if num2 % 2 == 0:
            if num2 > 0:
                suma = num1 + num2
                print("La suma es:", suma)
            else:
                print("El segundo número no es positivo.")
        else:
            print("El segundo número no es par.")
    else:
        print("El primer número no es positivo.")
else:
    print("El primer número no es par.")

File: P2A2 Algoritmos a Lenguajes/sumas_pares.cpp
//Emilia Cristina Prieto Henriquez
//ID 00580909
//Ing. de Animacion Digital

#include <iostream>
using namespace std;

int main() {
    int Num1, Num2, Suma;

    cout << "Ingrese el primer numero: ";
    cin >> Num1;

    if (Num1 % 2 == 0) 
    {
        
        if (Num1 > 0) 
        {
            cout << "Ingrese el segundo numero: ";
            cin >> Num2;

            if (Num2 % 2 == 0) 
            {
                
                if (Num2 > 0) 
                {
                    Suma = Num1 + Num2;
                    cout << "La suma es: " << Suma << endl;
                } else {
                    cout << "El segundo numero no es positivo." << endl;
                }
                
            } else 
            {
                cout << "El segundo numero no es par." << endl;
            }

        } else 
        {
            cout << "El primer numero no es positivo." << endl;
        }

    } else 
    {
        cout << "El primer numero no es par." << endl;
    }

    return 0;
}